example/otel-collector/main.go

```diff
@@ -88,8 +89,34 @@ func initProvider() (func(context.Context) error, error) {
 	// set global propagator to tracecontext (the default is no-op).
 	otel.SetTextMapPropagator(propagation.TraceContext{})
 
-	// Shutdown will flush any remaining spans and shut down the exporter.
-	return tracerProvider.Shutdown, nil
+	// Set up a metric exporter
+	metricExporter, err := otlpmetricgrpc.New(ctx, otlpmetricgrpc.WithGRPCConn(conn))
```

Comment:
For readability, I would split setup into 2 methods. One for tracing and one for metrics.